android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/PlayerContainerView.java

```diff
@@ -90,13 +91,15 @@ public void init() {
         audioCodecPriority.add("mp4a.a5");
         audioCodecPriority.add("mp4a.40");
 
+        boolean shouldEnableTunneledPlayback = Build.MANUFACTURER.contains("Google") || Build.MANUFACTURER.contains("Amazon");
```

Comment:
Have you tested this on our targeted devices (Chromecast with Google TV, all Fire TV devices)?

Reply:
I've tested this on my Chromecast with Google TV and it works. Haven't tested on Fire TV since I don't own one, but @CSeal showed me earlier that `react-native-device-info`'s `getManufacturerSync` method returns "Amazon" for him. That `getManufacturerSync` is a [wrapper around Android's `Build.MANUFACTURER`](https://github.com/react-native-device-info/react-native-device-info/blob/master/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java) - see screenshot.

![image](https://user-images.githubusercontent.com/48723873/192778516-f3639669-d9c0-4bdb-aa0c-9a0337c3dbcc.png)